src/css/app.sass

```diff
@@ -57,6 +58,7 @@ body
 
 body.body--light
   background: #fafafa
+  background-image: url(../assets/background.svg) !important
```

Comment:
is there a reason to use an SVG asset vs. a CSS gradient? if not, the gradient is much smaller size, though in that cae `MainLayout.vue` `:229` should be deleted and combined with the `body` gradient(s)

Reply:
Just to match the one we have on telos.net, it stretches with the screen height too.